keep/api/core/db.py

```
@@ -1061,12 +1061,15 @@
 ) -> list[Alert]:
     with Session(engine) as session:
         # Create the query
-        query = session.query(Alert)
+        query = (
+            session
+            .query(Alert)
```

Ruff (0.8.0)

`Alert` may be undefined, or defined from star imports

(F405)